packages/genesys-spark-components/src/components/beta/gux-side-panel/gux-modal-side-panel.scss

```diff
@@ -0,0 +1,20 @@
+@use 'mixins';
+
+dialog {
+  right: 0;
+  left: auto;
```

Comment:
Can you use logical properties here instead? 
inset-inline-start & inset-inline-end